python/ml4ir/base/config/parse_args.py

```diff
@@ -59,7 +59,7 @@ def define_args(self):
         self.add_argument(
             "--optimizer_key",
             type=str,
-            default="adam",
+            default="Adam",
```

Comment:
Change needed because we need to match exactly the name of the class at `tf.keras.optimizers`

Reply:
Can you update config keys?

Reply:
Yes. I need to do this.